crates/egui-wgpu/Cargo.toml

```diff
@@ -37,7 +37,7 @@ default = ["fragile-send-sync-non-atomic-wasm"]
 puffin = ["dep:puffin"]
 
 ## Enable [`winit`](https://docs.rs/winit) integration. On Linux, requires either `wayland` or `x11`
-winit = ["dep:winit"]
+winit = ["dep:winit", "winit/rwh_06"]
```

Comment:
Could do the same as eframe for consistency, although it's the same result in the end.

https://github.com/emilk/egui/blob/24cd711f5b9fce1e0ee28dbe518651b1a84899f8/crates/eframe/Cargo.toml#L149

Reply:
Looking like it's already enabled for eframe? 🤔